README.md

```diff
@@ -1,2 +1,14 @@
 # Quiz-App
-Project Quiz App for HYF
+
+A simple quiz application built with HTML, CSS, and JavaScript.
```

Comment:
It is a very good and pro practice to include a readme file in a project. Big, big kudos 👍 🏆